tests/sensors/test_base_sensor.py

```diff
@@ -502,3 +502,28 @@ def test_sensor_with_invalid_timeout(self):
             return_value=None,
             poke_interval=10,
             timeout=positive_timeout)
+
+    def test_sensor_with_exponential_backoff_off(self):
+        sensor = self._make_sensor(
+            return_value=None,
+            poke_interval=5,
+            timeout=60,
+            exponential_backoff=False)
+
+        started_at = timezone.utcnow() - timedelta(seconds=10)
+        self.assertEqual(sensor._get_next_poke_interval(started_at, 1), sensor.poke_interval)
+        self.assertEqual(sensor._get_next_poke_interval(started_at, 2), sensor.poke_interval)
```

Comment:
I'd rather we had the exact values here in the tests - this feels like it's just running the code again, but not actually testing the behaviour.

Reply:
`sensor.poke_interval` is an exact value `5`. The test is to make sure that between diff tries, the output is not changing.

Reply:
Oh yes, this one is fine then.